xmppserver/src/main/webapp/offline-messages.jsp

```diff
@@ -296,6 +308,67 @@
         </tbody>
         </table>
     </div>
+    <br/>
+    <div class="jive-contentBoxHeader">
+        <fmt:message key="offline.messages.clean.title" />
+    </div>
+    <div class="jive-contentBox">
+        <table cellpadding="3" cellspacing="0" border="0">
+            <tbody>
+                <tr valign="top">
+                    <td colspan="2" style="padding-top: 2em;">
+                        <fmt:message key="offline.messages.clean.description" />
+                    </td>
+                </tr>
+                <tr valign="top">
+                    <td width="1%" nowrap>
+                        <b>
+                            <label for="offlinecleaner">
+                                <fmt:message key="offline.messages.clean.label" />
+                            </label>
+                        </b>
+                    </td>
+                    <td width="99%">
+                        <input type="checkbox" name="offlinecleaner" id="offlinecleaner" size="5" <%= offlinecleaner.equalsIgnoreCase("true") ? "checked" : "" %> >
```

Comment:
Please prevent usage of `<%=` - style snippets. This can probably be replaced by `${offlinecleaner ? "checked" : ""}` if you put that value to the page context.

Reply:
changed to ${offlinecleaner ? "checked" : ""}